src/index.rs

```diff
@@ -23,6 +23,15 @@ pub struct TagOptions {
     tag_case_sensitive: bool,
 }
 
+impl Default for TagOptions {
+    fn default() -> Self {
+        Self {
+            tag_separator: Option::None,
+            tag_case_sensitive: false,
+        }
+    }
+}
```

Comment:
Looks good!
BTW, why `Option::None` instead of just `None`?

Reply:
:+1: